book/src/faq.md

```diff
@@ -1,5 +1,18 @@
 # FAQ
 
+## Why aren't property values updating for my service that doesn't notify changes?
+
+A common issue might arise when using zbus and your service's property values aren't updating. 
+This is due to zbus' default caching policy, which updates the value of a property only when 
+a change is signaled, primarily to minimize latency and optimize client request performance. 
+By default, if your service does not emit change notifications, the property values will not 
+update accordingly. 
+
+However, you can explicitly bypass this caching mechanism by using the 
+`#[dbus_proxy(property(emits_changed_signal = "false"))]` annotation. 
```

Comment:
Similarly here, we're assuming that it's a specific proxy type generated by `dbus_proxy`. One can have the same issue with `Proxy`. We should mention that [`ProxyBuilder::uncached_properties` method](https://docs.rs/zbus/3.13.1/zbus/struct.ProxyBuilder.html#method.uncached_properties) can be used when dealing with `Proxy`.